scripts/run.py

```diff
@@ -256,6 +256,8 @@ def parse_args():
 				cam_matrix = f["transform_matrix"]
 				testbed.set_nerf_camera_matrix(np.matrix(cam_matrix)[:-1,:])
 				outname = os.path.join(args.screenshot_dir, os.path.basename(f["file_path"]))
+				if not outname.endswith('.png'):
```

Comment:
Thank you for the constribution! Could you replace this check with `if not os.path.splitext(outname)[1]:`?

Reasoning: some datasets may use `.jpg` or other extensions. We do not want to append `.png` in this case, but rather use tha datasets expected format.

Reply:
good call!  yeah I guess it's fair that this code can work for more arbitrary input